docs/UserGuide.md

```diff
@@ -156,13 +156,24 @@ Format: `delete INDEX`
 
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all persons or tags : `clear [t/TAG]`
+### Removing a person's field/attribute : `remove`
+
+Removes the specific field value based on the given input
+
+Format: `remove INDEX [n/NAME] [p/PHONE] [p/GITHUBPROFILE] [e/EMAIL] [a/ADDRESS] [l/LANGUAGE] [t/TAG]...`
+
+* Removes field value of person at the specific `INDEX`.
+* Removes all the corresponding field value in respect of the `KEYWORD`.
+  * e.g. `l/Java` will remove `Java` in person’s **Language** field.
+* Removes the entire field value when that field has no keyword provided.
+  * e.g. `t/` will remove all the tags associate to the person.
+
+### Clearing all persons or tags : `clear`
```

Comment:
Not sure if you'd like to include a
Format: `clear` for consistencies sake

Reply:
The Format is at line 180

Reply:
I see, missed that, my bad